src/Microsoft.VisualStudio.SlowCheetah/Transformer/XmlTransformer.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.VisualStudio.SlowCheetah
     /// </summary>
     public class XmlTransformer : ITransformer
     {
+        private const string XdtAttributeName = "xmlns:xdt";
+        private const string XdtAttributeValue = "http://schemas.microsoft.com/XML-Document-Transform";
```

Comment:
There is some other resources that can be pulled out, like the Json default transform file. It is just `"{ }"`